tasks/main.yml

```diff
@@ -29,15 +29,18 @@
   register: uname_output
 
 - name: Install linux-image-extra-* packages to enable AuFS driver
-  apt: pkg={{ item }} state=present
+  apt:
+    pkg: "{{ item }}"
+    state: present
+    update_cache: yes
   with_items:
     - linux-image-extra-{{ uname_output.stdout }}
     - linux-image-extra-virtual
   when: ansible_distribution_version|version_compare(14.04, '>=')
 
 - name: Run dmsetup for Ubuntu 16.04
   command: dmsetup mknodes
-  when: dmsetup_result
+  when: dmsetup_result.get("skipped", False) != True
```

Comment:
When you rebase you'll see that this is already addressed so I think you can just remove this.